src/main/java/io/pakland/mdas/githubstats/infrastructure/github/controller/GitHubUserOptionController.java

```diff
@@ -120,13 +123,15 @@ private void fetchCommitsFromPullRequest(Repository repository, PullRequest pull
         //
         try {
             List<Commit> commitList = new FetchCommitsFromPullRequest(
-                commitExternalRepository)
-                .execute(repository.getOwnerLogin(), repository.getName(),
+                    commitExternalRepository)
+                    .execute(repository.getOwnerLogin(), repository.getName(),
+                            pullRequest.getNumber());
+            //TODO: Fetch PR Reviews.
+            
+            List<Comment> commentList = new FetchCommentsFromPullRequest(
+                    commentExternalRepository).execute(repository.getOwnerLogin(), repository.getName(),
```

Comment:
refactor suggestion: preserve whole repository object

Reply:
Yep, that's what I did in #144, but @manerow handn't yet seen it